### ISSUE TITLE
DPL-590-2 Only display tag layout templates that are enabled

### ISSUE THREAD

**Issue by sdjmchattie** [sanger/gatekeeper#245]
**User story**
As a lab user, I want to not be shown out-of-date tag layout templates when I'm registering pre stamped tag lots so that I don't pick the wrong one with a similar name.

**Who are the primary contacts for this story**
sm49, as28, sg21, ag25

**Who is the nominated tester for UAT**
ag25

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] The entry for TSsc384-PCR1-nCoV-2019/V3/B and TSsc384-PCR2-nCoV-2019/V3/B as options for the tag layout template no longer appear on the "Register IDT Tags Lot" screen, the "Register Pre Stamped Tags Lot" screen or the "Register Pre Stamped Tags - 384 Lot" screen.

**Dependencies**
This story is blocked by the following:
- https://github.com/sanger/sequencescape/issues/3721

This story blocks the completion of the following:
- https://github.com/sanger/limber/issues/1204

**Comment by sdjmchattie** [comment]
No longer needed as no change was required
